fabric/couchdb/local.ini

```diff
@@ -0,0 +1,88 @@
+; CouchDB Configuration Settings

```

Comment:
Can we filter out these couchdb things again?  Maybe we should remove it from fnb?